sdk/tests/deqp/framework/common/tcuFloat.js

```diff
@@ -596,37 +633,81 @@ tcuFloat.numberToFloat11 = function(value) {
     return tcuFloat.newFloat11(value).bits();
 };
 
-tcuFloat.float11ToNumber = function(float11) {
-    var x = tcuFloat.newDeFloatFromParameters(float11, tcuFloat.description11);
-    return x.getValue();
-};
+tcuFloat.float11ToNumber = (function() {
+    var x = tcuFloat.newDeFloatFromParameters(0, tcuFloat.description11);
+    return function(float11) {
+        x.deFloatParametersNumber(float11);
+        return x.getValue();
+    };
```

Comment:
Yes, exactly. Specifically, these functions were creating new deFloats, which in turn did very expensive TypedArray allocations (sometimes multiple times) which were then immediately discarded after producing the value. By only allocating the deFloat once we avoid TypedArray allocation churn.